frontend/src/components/RecommendedRecipes.js

```diff
@@ -70,45 +70,29 @@ class RecommendedRecipes extends Component {
     this.state = {
       isLoading: true,
       isRedirect: false,
-      // hard-coded data
-      recipes: [
-        {
-          name: "Dish 1",
-          time: "18 min",
-          level: "Easy",
-          calories: "383 kcal",
-          ingredients: ["broccoli", "tomato"],
-          instructions: ["step1: broccoli", "step2: tomato"],
-        },
-        {
-          name: "Dish 2",
-          time: "25 min",
-          level: "Hard",
-          calories: "243 kcal",
-          ingredients: ["egg", "tomato"],
-          instructions: ["step1: egg", "step2: tomato"],
-        },
-        {
-          name: "Dish 3",
-          time: "20 min",
-          level: "Easy",
-          calories: "342 kcal",
-          ingredients: ["egg", "pasta"],
-          instructions: ["step1: egg", "step2: pasta"],
-        },
-      ],
+      recipes: [],
     };
   }
 
+  // retrieves recommended recipes from the back end 
   componentDidMount() {
-    // post request to get recommended recipes
-    this.setState({ isLoading: false });
+    const request = new Request("/api/find-recipes", {
```

Comment:
find-recipes is what I was looking for. Make sure your backend looks for this in the request.

Reply:
The web servlet FindRecipes has a url pattern `/api/find-recipes`. I can create a base url field `/api `that will be applied to every request, so that this line would be `new Request("/find-recipes"`. I will do it in the next PR